cg/cli/upload/genotype.py

```diff
@@ -1,10 +1,13 @@
 """Code for uploading genotype data via CLI"""
 import click
+import logging
 
 from cg.meta.upload.gt import UploadGenotypesAPI
 
 from .utils import _suggest_cases_to_upload
```

Comment:
This seems a bit surprising, importing an "_"-function (private), maybe we should rename it?

Reply:
Sure, I can change that. I have no idea why did it from the beginning though

Reply:
fixed